src/evict/evict_lru.c

```diff
@@ -574,52 +558,70 @@ __evict_pass(WT_SESSION_IMPL *session)
 		 * workers, it must service the urgent queue in case all
 		 * application threads are busy.
 		 */
-		if (cache->evict_empty_score < WT_EVICT_EMPTY_SCORE_CUTOFF ||
+		if (cache->evict_empty_score < WT_EVICT_SCORE_CUTOFF ||
 		    (!WT_EVICT_HAS_WORKERS(session) &&
 		    !__evict_queue_empty(cache->evict_urgent_queue)))
 			WT_RET(__evict_lru_pages(session, true));
 
+		if (cache->pass_intr != 0)
```

Comment:
I agree that sweep could cause interruptions in that case...

We use `pass_intr` for a couple of different things, but I think it's mostly to encourage the eviction server to release the handle list lock.

Should we discuss alternatives to this change?